pkg/volume/hostpath/host_path_test.go

```diff
@@ -172,7 +172,14 @@ func TestProvisioner(t *testing.T) {
 	if err != nil {
 		t.Errorf("Failed to make a new Provisioner: %v", err)
 	}
-	pv, err := creator.Provision(nil, nil)
+
+	hostPathCreator, ok := creator.(*hostPathProvisioner)
+	if !ok {
+		t.Error("Not a hostPathProvisioner")
```

Comment:
should be fatal?  The other errors above should also be fatal I think

Reply:
> should be fatal? The other errors above should also be fatal I think

done